src/pages/cb-banks/Windows/CbBanksWindow.js

```diff
@@ -5,21 +5,28 @@ import CustomTabPanel from 'src/components/Shared/CustomTabPanel'
 import { InterfacesForm } from 'src/components/Shared/InterfacesForm'
 import { ResourceIds } from 'src/resources/ResourceIds'
 
-const CbBanksWindow = ({ height, labels, maxAccess, recordId }) => {
+const CbBanksWindow = ({ labels, maxAccess, recordId, height }) => {
   const [activeTab, setActiveTab] = useState(0)
 
   const [store, setStore] = useState({
     recordId: recordId || null,
     name: null
   })
+  const [editMode, setEditMode] = useState(!!recordId)
```

Comment:
can use edit mode as constant instead state

Reply:
no, i am using setEditMode in the form

Reply:
why no? we stoped using setEditMode